### DISCUSSION TITLE
공격 스탯에 관한 식 부족

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/HumanSurvival#43] [Q&A]
아래에 관한 식이 없습니다.
[스킬 시스템](https://github.com/SUST22Unity/HumanSurvival/wiki/%EC%8A%A4%ED%82%AC-%EC%8B%9C%EC%8A%A4%ED%85%9C#%EC%8A%A4%ED%83%9D%EB%B3%84-%EC%8B%9D)
* 투사체 속도
* 지속 시간
* 공격범위
* 쿨타임

**Comment by rapa1130** [comment]
[문서 링크](https://github.com/SUST22Unity/HumanSurvival/wiki/%EC%8A%A4%ED%82%AC-%EC%8B%9C%EC%8A%A4%ED%85%9C#%EC%8A%A4%ED%83%9D%EB%B3%84-%EC%8B%9D)

> 스택별 식
> 피해량, 투사체 속도, 지속 시간, 공격 범위, 쿨타임
> 이 스탯들은 퍼센테이지 수치들로 특정 스킬에 해당하는 값에 캐릭터 스탯과 퍼센테이지 연산을 하면 된다.
> 
> 피해량을 예로 들어보자.
> 
> 계산된 피해량 = 해당 스킬 피해량 * (1 + 피해량스탯 / 100)
> 
> 예를 들면 스킬 피해량 20 그리고 피해량 스탯이 50% 라면
> 
> 피해량 = 20 * (1 + 50 / 100) = 30 
> 

피해량 뿐만 아니라 다른 수치들에 대해서도 같은 식을 적용하면 됩니다.

    계산된 투사체 속도 = 해당 스킬 투사체 속도 * (1 + 투사체 속도 스탯 / 100)
    계산된 지속 시간 = 해당 스킬 지속 시간 * (1 + 지속 시간 스탯 / 100)
    계산된 공격 범위 = 해당 스킬 공격 범위 * (1 + 공격 범위 스탯 / 100)
    계산된 쿨타임 = 해당 스킬 피해량 * (1 + 쿨타임 스탯 / 100)
